lib/ui/src/app.js

```diff
@@ -41,35 +41,45 @@ const View = styled.div({
   width: '100vw',
 });
 
-const App = React.memo(({ viewMode, layout }) => {
+const App = React.memo(({ viewMode, layout, size: { width, height } }) => {
   const props = createProps();
 
+  let content;
+
+  if (!width || !height) {
+    content = (
+      <div>
+        {width} x {height}
+      </div>
+    );
+  }
+  if (width < 600) {
+    content = <Mobile {...props} viewMode={viewMode} options={layout} />;
+  }
+
+  content = <Desktop {...props} viewMode={viewMode} options={layout} {...{ width, height }} />;
```

Comment:
won't this always override the mobile layout above?

Reply:
fixed